bithubph/bithub-fe
# #172103237 Setup react styleguidist

#### What does this PR do?
- This PR sets up a documentation styleguide for every component

#### Description of tasks to be completed?
- install ` --save-dev react-styleguidist`
- add this command into package.json script for config` "scripts": {
     "styleguide": "styleguidist server",
    "styleguide:build": "styleguidist build"
  }`

#### How should this be manually tested?
  - Clone the repo with `clone https://github.com/bithubph/bithub-fe.git`
  - Checkout to branch `ft-setup-react-styleguidist-172103237`
  - Run application `npx styleguidist server`
  - The result will appear on the terminal:
          `i ｢wds｣: Project is running at http://localhost:6060/`
          `i ｢wds｣: webpack output is served from undefined`
          `i ｢wds｣: Content not from webpack is served from C:\develop\react-apps\bithub-fe`
         `You can now view your style guide in the browser:`

             `Local:            http://localhost:6060/`
             `On your network:  http://192.168.56.1:6060/`

              `DONE  Compiled successfully!`

#### Any background context you want to provide?
- You would need npm or yarn to run this application

#### What are the relevant pivotal tracker stories?
- PT Story link - [#172103237](https://www.pivotaltracker.com/n/projects/2440925)


#### Screenshots (if appropriate)
![Screenshot (11)](https://user-images.githubusercontent.com/27733455/78371770-50eb7400-75c0-11ea-8caf-262527d604a2.png)
